e2e/test/scenarios/embedding/embedding-questions.cy.spec.js

```diff
@@ -306,7 +306,7 @@ describeEE("scenarios > embedding > questions > downloads", () => {
           "Trying to prevent downloads via query params doesn't have any effect",
         );
         cy.url().then(url => {
-          cy.visit(url + "&hide_download_button=true");
+          cy.visit(url + "&downloads=false");
```

Comment:
Not an e2e one no, the "old" behaviour is only tested in the util function, to test this on e2e it would mean having to write at least 4 tests (public question, static question, public dashboard, static question). Do you think they're worth it?
hide_downloads+downloads mixed is tested already in the util function, and the `downloadsEnabled` flag is tested at the unit level also in the components IIRC